### DISCUSSION TITLE
How to use v2ray docker container?

### DISCUSSION THREAD

**Discussion by skorphil** [v2fly/v2ray-core#2860] [Q&A]
Hi, i'm trying to use https://hub.docker.com/r/v2fly/v2fly-core and container exits after launching. 
At first i think that's because of M1 hardware https://github.com/v2fly/v2ray-core/discussions/2859 but on ubuntu on vps i got same result - container exits immediately after been launched. 

How do i suppose to use it? There are no guides on how to use containerized version

**Comment by AkinoKaede** [comment]
Here is some usage for this image. https://github.com/v2fly/docker/blob/master/README.md
